CLUE_Metronome/clue_metronome.py

```diff
@@ -0,0 +1,209 @@
+import time
+import array
+import math
+import board
+from audiocore import RawSample
+import audiopwmio
+import displayio
+import terminalio
+from adafruit_display_text import label
+from adafruit_display_shapes.rect import Rect
+from adafruit_display_shapes.circle import Circle
+from adafruit_clue import clue
+
+blink_dot = False  # optional blinking dot on accent
+blink_light = True  # optional flashing backlight on accent
+tempo = 120  # in bpm
+print("BPM: {}".format(tempo))
+time_signature = 4  # Beats per measure
+BEEP_DURATION = 0.05
+delay = 60 / tempo
+
+# constants for sine wave generation
+SIN_LENGTH = 100  # more is less choppy
+SIN_AMPLITUDE = 2 ** 15  # 0 (min) to 32768 (max)
+SIN_OFFSET = 32767.5  # for 16bit range, (2**16 - 1) / 2
+DELTA_PI = 2 * math.pi / SIN_LENGTH  # happy little constant
+
+sine_wave = [
+    int(SIN_OFFSET + SIN_AMPLITUDE * math.sin(DELTA_PI * i)) for i in range(SIN_LENGTH)
+]
+tones = (
```

Comment:
can you use simpleio tone instead of making sine waves?

Reply:
Ah, I'll try that! My head was stuck back in TTY machine tone land...

Reply:
@ladyada this uses simpleio tone now.